rulebook-core/src/main/java/com/deliveredtechnologies/rulebook/model/RuleBook.java

```diff
@@ -46,4 +47,7 @@ default void defineRules() { }
    * @return  true if any Rules have been defined in the RuleBook, otherwise false.
    */
   boolean hasRules();
+  
+  Map<String, Map<Long, RuleStatus>> getAudit();
```

Comment:
Why do you have a RuleBook returning the audit? An Auditor decorates the RuleBook.

Reply:
hi @Clayton7510 yes I really struggled with this , and agree with your comments on the separation. I went around in circle but just couldn't see a way for RuleBookAutitor to inspect a RuleBook auditor results. I'll try again this week and see if there is any other solutions that don't mean chanign the interface

Reply:
hi @Clayton7510 not sure why I got myself into changing the RuleBook , think I must have spent too long on this and got myself into knots. Looking at a "fresher" set of eye , I think I can manage the same outcome without changing the RuleBook interface , made a change which only really changes the RuleBookAuditor and the test (including the new ones I added) all pass ok , appreciate if you could have another look , thanks